### ISSUE TITLE
Accessing over ethernet port is disabled by default

### ISSUE THREAD

**Issue by daniellandau** [piwebcam/PiWebcam#34]
The default iptables rules disable access over Ethernet. Is there a reason for that? In my opinion in many cases it's simpler and easier to run an Ethernet cable to the Pi and you get faster and more reliable communication too.

**Comment by daniellandau** [comment]
I can make the pull request if there's no reason to disable Ethernet access.

**Comment by user2684** [comment]
Definitely not intended to be like that! If you want to submit a PR request cool, otherwise I can fix it no problem

**Comment by iaglet** [comment]
While it's not desirable, this may be a default behavior from the OS. "disable ethernet when wifi active". Though it's interesting there is also no way in the interface to disable the wifi. I am not even sure that entering bogus credentials would work around this.

**Comment by user2684** [comment]
Yeah, you have a point this project was intended at the beginning to be entirely managed via wifi but ethernet should be something to be considered better. I'll pick my brain on it. Thanks!
